.circleci/config.yml

```diff
@@ -1383,9 +1465,11 @@ workflows:
       - spm-release-build
       - spm-release-build-xcode-14
       - spm-release-build-xcode-15
+      - spm-release-build-xcode-16
```

Comment:
I believe this is the same as `spm-release-build` since that one runs on the default (16)